packages/sources/dar/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,138 @@
+import * as process from 'process'
+import { AddressInfo } from 'net'
+import {
+  mockWebSocketProvider,
+  mockPriceWebSocketServer,
+  createAdapter,
```

Comment:
Just a general comment, the test file here is hard to read. If there's anything you can do to format or abstract some of the logic, that may help.